portfolio/src/main/webapp/chartScript.js

```diff
@@ -0,0 +1,26 @@
+google.charts.load('current', {'packages':['line']});
+google.charts.setOnLoadCallback(drawStateChart);
+
+function drawStateChart() {
+  fetch('/chart-data').then(response => response.json())
+  .then((covidCases) => {
+    buildChart(covidCases[0], "state-chart-div");
+    buildChart(covidCases[1], "county-chart-div");
+  });
+}
+
+function buildChart(dataMap, divPath) {
+  const data = new google.visualization.DataTable();
+  data.addColumn('date', 'Date');
+  data.addColumn('number', 'Cases');
+  Object.keys(dataMap).forEach((day) => {
+      data.addRow([new Date(day), dataMap[day]]);
+    });
```

Comment:
You could use the Object.entries to improve readability:
for (let [key, value] of Object.entries(object1)) {
  console.log(`${key}: ${value}`);
}